src/services/OgSelectionManager.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Core\Entity\EntityReferenceSelection\SelectionPluginManager.
+ */
+
+namespace Drupal\og\services;
+
+use Drupal\Component\Plugin\FallbackPluginManagerInterface;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityReferenceSelection\SelectionPluginManager;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\DefaultPluginManager;
+use Drupal\og\Og;
+
+/**
+ * Plugin type manager for Entity Reference Selection plugins.
+ *
+ * @see \Drupal\Core\Entity\Annotation\EntityReferenceSelection
+ * @see \Drupal\Core\Entity\EntityReferenceSelection\SelectionInterface
+ * @see plugin_api
+ */
+class OgSelectionManager extends SelectionPluginManager  {
```

Comment:
Not sure we really need this service? This could easily just be a method on the `OgSelectionHandler`? This is kind of just something it uses internally?